### ISSUE TITLE
Make read more link simpler

### ISSUE THREAD

**Issue by chriswallace** [ChurchThemes-WP/uplifted-theme#99]
http://cl.ly/image/0g053v290N2N

No button style for that one.

**Comment by angela-crist** [comment]
@chriswallace What needs to be complete for this?  Just add style?

**Comment by garand** [comment]
Nah, he wants to remove the button style and make it look just like a text link.

**Comment by garand** [comment]
I'll do this sometime this week.
